RevenueCatUI/PaywallView.swift

```diff
@@ -26,7 +26,8 @@ public struct PaywallView: View {
 struct PaywallView_Previews: PreviewProvider {
 
     static var previews: some View {
-        PaywallView(offering: TestData.offering, paywall: TestData.paywall)
+        let offering = TestData.offeringWithIntroOffer
+        PaywallView(offering: offering, paywall: offering.paywall!)
```

Comment:
might as well do an if let here and have a text explaining that there's no offering so the preview doesn't ugly-crash? 

I.e.: 

```swift
        let offering = TestData.offeringWithIntroOffer
        if let paywall = offering.paywall {
            PaywallView(offering: offering, paywall: paywall)
        } else {
            Text("Preview not correctly set up! The offering's paywall need to be non-nil")
        }
```

Reply:
👍🏻  I guess debugging crashes in previews is annoying, so yeah.

Reply:
Done in the next PR.